pkg/kubelet/cm/cpuset/cpuset_test.go

```diff
@@ -343,4 +345,20 @@ func TestParse(t *testing.T) {
 			t.Fatalf("expected string \"%s\" to parse as [%v] (got [%v])", c.cpusetString, c.expected, result)
 		}
 	}
+
+	negativeTestCases := []string{
+		// Non-numeric entries
+		"nonnumeric", "non-numeric", "no,numbers", "0-a", "a-0", "0,a", "a,0", "1-2,a,3-5",
+		// Incomplete sequences
+		"0,", "0,,", ",3", ",,3", "0,,3",
```

Comment:
nit: these are all supported by the kernel, since it just ignores extra commas. Not sure if we should care tho.

Reply:
This cpuset format is consumed by others too, like kubelet and cri-o, so perhaps it's worth being more picky.